baybe/recommenders/base.py

```diff
@@ -31,7 +36,8 @@ def _select_candidates_and_recommend(
 
     Args:
         searchspace: The search space.
```

Comment:
Need to adjust the docstring here since this is now no longer a search space but a subspace.